lib/noid/minter.rb

```diff
@@ -93,7 +95,16 @@ def counters
     end
 
     def dump
-      { state: true, seq: @seq, seed: @seed, template: template.template, counters: Marshal.load(Marshal.dump(counters)) }
+      {
+        template: template.template,
+        counters: Marshal.load(Marshal.dump(counters)),
```

Comment:
Because we were `load`ing before. I think the idea was to deep dup the counters.